main/zoe/api/contract-support/ratio-math.md

```diff
@@ -62,8 +62,8 @@ A ratio has these restrictions:
 - The denominator cannot be 0. 
```

Comment:
Typo in line 3

Reply:
Also, in line 8, records are a type of object that has no methods and is pass by copy, so "records, not objects" doesn't make sense